src/pages/MyPage/MyPage.js

```diff
@@ -0,0 +1,256 @@
+  },
+  {
+    id: '5',
+    text: '추천합니다.',
+    sitter: '오현아',
+    date: '2022-05-01 17:28',
+  },
+];
+
+const reservationData = [
+  {
+    id: 1,
+    sittername: '장형원',
+    reservationStartDate: '2022-02-01',
+    reservationEndDate: '2022-02-03',
+    petcount: 2,
+    cancelStatus: true,
+    reservationState: 'loading',
+  },
+  {
+    id: 2,
+    sittername: '이현정',
+    reservationStartDate: '2022-12-02',
+    reservationEndDate: '2022-12-03',
+    petcount: 1,
+    cancelStatus: true,
+    reservationState: 'loading',
+  },
+  {
+    id: 3,
+    sittername: '유동혁',
+    reservationStartDate: '2021-07-13',
+    reservationEndDate: '2021-07-15',
+    petcount: 3,
+    cancelStatus: true,
+    reservationState: 'passed',
+  },
+  {
+    id: 4,
+    sittername: '이수광',
+    reservationStartDate: '2022-10-20',
+    reservationEndDate: '2022-10-23',
+    petcount: 2,
+    cancelStatus: true,
+    reservationState: 'passed',
+  },
+];
+
+const MyPage = () => {
+  const [status, setStatus] = useState(true);
+  const [reviews, setReviews] = useState(reviewData);
+  const [reserve, setReserve] = useState(reservationData);
+
+  const reserveNow = reserve.filter(
+    state => state.reservationState === 'loading'
+  );
+  const reservePassed = reserve.filter(
+    state => state.reservationState === 'passed'
+  );
+
+  const goDelete = id => {
+    if (window.confirm('리뷰를 삭제하시겠습니까?')) {
+      setReviews(reviews.filter(review => review.id !== id));
+      alert('리뷰가 삭제되었습니다.');
+    }
+  };
+
+  const cancelRes = id => {
+    const reserveIdx = reserve.findIndex(reserve => reserve.id === id);
+    if (reserve[reserveIdx].cancelStatus) {
+      if (window.confirm('예약을 취소 요청하시겠습니까?')) {
+        alert('요청되었습니다.');
+        const newReserve = [...reserve];
+        newReserve[reserveIdx].cancelStatus = false;
+        setReserve(newReserve);
+      }
+    } else {
+      alert('이미 취소요청되었습니다. 변경하시려면 고객센터로 문의바랍니다.');
+    }
+  };
+
+  return (
+    <MyPageWrapper>
+      <MyPageWrapperDiv>
+        <MyPageTitle>예약내역</MyPageTitle>
+        <ReservationState>
+          <StatusBtnWrapper>
+            <StatusNowBtn status={status} onClick={() => setStatus(true)}>
+              진행중인 예약
+            </StatusNowBtn>
+            <StatusPassedBtn status={status} onClick={() => setStatus(false)}>
+              지난 예약
+            </StatusPassedBtn>
+          </StatusBtnWrapper>
+          {status ? (
+            <MyReservation cancelRes={cancelRes} reserveNow={reserveNow} />
+          ) : (
+            <PassedReservation reservePassed={reservePassed} />
+          )}
+        </ReservationState>
+        <MyReviewWrapper>
+          <MyReviewTitle>내가 쓴 리뷰</MyReviewTitle>
+          <ReviewState>
+            {reviews.length === 0 ? (
+              <NoneReview>내가 쓴 리뷰가 없습니다</NoneReview>
+            ) : (
+              reviews.map((review, idx) => (
+                <MyReview
+                  key={idx}
+                  goDelete={goDelete}
```

Comment:
여기서 아예 해당 id를 보내줄 수 있습니다.
```suggestion
                  goDelete={() => goDelete(review.id)}
```